core-management/src/main/java/org/wildfly/extension/core/management/ConfigurationChangeResourceDefinition.java

```diff
@@ -109,32 +97,49 @@ public void registerAttributes(ManagementResourceRegistration resourceRegistrati
         resourceRegistration.registerReadWriteAttribute(MAX_HISTORY, null, new MaxHistoryWriteHandler(ConfigurationChangesCollector.INSTANCE));
     }
 
+
     private static class ConfigurationChangeResourceAddHandler extends AbstractAddStepHandler {
-        public ConfigurationChangeResourceAddHandler() {
-            super(MAX_HISTORY);
+        private ConfigurationChangeResourceAddHandler() {
+            super(CONFIGURATION_CHANGES_CAPABILITY, MAX_HISTORY);
```

Comment:
You pass CONFIGURATION_CHANGES_CAPABILITY to the ConfigurationChangeResourceDefinition constructor so you don't need to pass it to the add handler constructor.